main/system.h

```diff
@@ -56,7 +56,7 @@
     #define UHF_SCI CSP_SCI  //UART2
 #if defined(IS_3U)
     #define GPS_SCI sciREG1  //UART3
-#else if defined(IS_2U)
+#elif defined(IS_2U)
```

Comment:
I see your point about style. Does this change anything? Just curious

Reply:
#else and #elif have different semantics. The prepocessor warning was "extra characters ignored". The extra characters that are ignored are "if defined(IS_2U)", i.e. the body of the else wasn't actually conditional on IS_2U being defined.